### PR TITLE
LL 6326 - LLM add twitter banner

### PR DESCRIPTION
**Pull request by klambert-ledger**
Adding a Twitter banner inside the Platform Catalog.
Banner is closable and hashtag pressable to go to a pre filled twitter form (through the browser or the Twitter App).

### Screenshots
<details>
  <summary>iOS 📸 </summary>

Simulator Screen Shot - iPhone 11 - 2021-07-07 at 16 53 10
</details>

### Type

Feature

### Context

[LL-6326](https://ledgerhq.atlassian.net/browse/LL-6326)

### Parts of the app affected / Test plan

Platform/Catalog
Translations

### CODE REVIEW

**Review by IAmMorrow — DISMISSED** [review]
Good first contribution ! 🥳 
Welcome to the team !

File: src/screens/Platform/CatalogBanner.js [hunk 1]
```diff
@@ -34,15 +36,25 @@ const styles = StyleSheet.create({
     borderRadius: 4,
     padding: 16,
     marginBottom: 16,
+    overflow: "hidden",
+    position: "relative",
   },
   title: {
     fontSize: 18,
-    marginBottom: 16,
+    marginBottom: 7,
```
Comment (juan-cortes) [inline src/screens/Platform/CatalogBanner.js:44]: We _try_ to follow 2,4,8,12,16,32 for distance so this should be an 8 even if on figma it's a 7, but 🤷🏼‍♂️

**Review by juan-cortes — DISMISSED** [review]
lgtm

File: src/screens/Platform/CatalogBanner.js (hunk 1 above)
Comment (klambert-ledger) [reply src/screens/Platform/CatalogBanner.js:44]: Yes that was the value on figma. I'll fix this right now ! 👍

**Review by IAmMorrow — APPROVED** [review]

**Review by lruggiero-ledger — DISMISSED** [review]

**Review by ychen-ledger — DISMISSED** [review]
Validated on ios side: 

- Iphone 6s 
- simulator iphone 11
- Nano X

**Comment by klambert-ledger** [comment]
Just moved the url to config to fit more with the review made on LLD (https://github.com/LedgerHQ/ledger-live-desktop/pull/3969) with the same feature. 

Sry @ychen-ledger if you have to retest it again 😞 (you really shouldn't have to, I just moved a variable and tested it before pushing it).

**Review by ychen-ledger — COMMENTED** [review]
@klambert-ledger No worries, feel free to ping me if any further test needed😄.

**Review by achabrolin-ledger — APPROVED** [review]
lgtm

**Pull request merged**